### PR TITLE
FastQC

### PR DESCRIPTION
**Pull request by tgaspe**
## Description

Creating FastQC component.

## Issue ticket number

Closes #xxxx 

## Checklist before requesting a review

- [x] I have performed a self-review of my code

- [x] Conforms to the [Contributing guidelines](https://github.com/viash-hub/base/blob/main/CONTRIBUTING.md)

- [x] Proposed changes are described in the [CHANGELOG.md](https://github.com/viash-hub/base/blob/main/CHANGELOG.md)

- [x] I have tested my code with `viash ns test --parallel -q <name or namespace>`

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [x] New functionality
  - [ ] Major changes
  - [ ] Minor changes
  - [ ] Documentation
  - [ ] Bug fixes

### CODE REVIEW

File: src/_authors/theodoro_gasperin.yaml
[the remark this answers is not in the document]
Comment (tgaspe) [reply src/_authors/theodoro_gasperin.yaml:1]: Thanks ;)

File: src/fastqc/config.vsh.yaml [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+          mode.
+
+      - name: --java
+        alternatives: -j
+        type: file
+        description: |
+          Provides the full path to the java binary you want to use to
+          launch fastqc. If not supplied then java is assumed to be in
+          your path.
+        example: file
```
Comment (rcannood) [inline src/fastqc/config.vsh.yaml:87]: Remove this; Java should be installed in the docker image

**Review by rcannood — CHANGES_REQUESTED** [review]

**Review by rcannood — CHANGES_REQUESTED** [review]
ci tests are failing (but error is not caught properly). Can you take a look?

**Comment by tgaspe** [comment]
Hi Robrecht,

I am not sure I understand which tests are failing?

**Comment by tgaspe** [comment]
I see, thanks.

File: src/fastqc/config.vsh.yaml [hunk 2]
```diff
@@ -0,0 +1,182 @@
+name: fastqc
+namespace: fastqc
```
Comment (rcannood) [inline src/fastqc/config.vsh.yaml:2]: No need for a namespace if there is just one command

```suggestion
name: fastqc

**Review by rcannood — APPROVED** [review]
LGTM!

**Pull request merged**